src/services/deno_service.ts

```diff
@@ -79,12 +49,62 @@ export default class DenoService {
   }
 
   /**
-   * @description
-   * Get JSON file of 3rd party modules on deno.land
+   * Fetches the description for a module
+   *
+   *     await getThirdPartyDescription("drash"); // "A REST microframework ..."
+   *
+   * @param importedModuleName - The imported module in which we want to get the description for
    *
-   * @returns {DenoLandDatabase}
+   * @returns The description
    */
-  public static getDenoLandDatabase(): DenoLandDatabase {
-    return denoLandDatabase;
+  public static async getThirdPartyDescription(
+    importedModuleName: string,
+  ): Promise<string> {
+    const res = await fetch(
+      "https://api.deno.land/modules?query={module:" + importedModuleName +
+        "}&limit=1",
+    );
+    const json: {
+      success: boolean;
+      data: {
+        total_count: number;
+        results: Array<{
+          name: string;
+          description: string;
+          star_count: number;
+          search_score: number;
+        }>;
+      };
+    } = await res.json();
+    const description = json.data.results[0].description;
+    return description;
+  }
+
+  /**
+   * Fetches the owner and repository name, for the given module
+   *
+   *     await getThirdPartyRepoAndOwner("drash"); // "drashland/deno-drash"
+   *
+   * @param importedModuleName - The imported module in which we want to get the repository for on github
+   *
+   * @returns The owner and repo name, eg "<owner>/<repo>"
+   */
+  public static async getThirdPartyRepoAndOwner(
+    importedModuleName: string,
+  ): Promise<string> {
+    const latestRelease = await DenoService.getLatestThirdPartyRelease(
+      importedModuleName,
+    );
+    const res = await fetch(
+      "https://cdn.deno.land/" + importedModuleName + "/versions/" +
```

Comment:
We should place the Deno URLs in a const at the top of the file so we can change them faster. Nit picky I know, but that's how deno is doing it and I feel like it's bound to change.

Reply:
so like:
```typescript
const denoVersionUrl = (name: string) => {
    return "https://cdn.deno.land/" + name + ...
}
const denoDescriptionUrl = (name: string) => {
    ...
}
...
```
would be improved but kind of like that?

Reply:
```
const DENO_CDN_URL = "https://cdn.deno.land/";
const res = await fetch(
      DENO_CDN_URL + importedModuleName + "/versions/" +
```

Reply:
https://github.com/drashland/dmm/pull/47/commits/bce63088d6a520b42a4ed576ff53425e68b1cb06, attached them to the class instead if that's cool? But good idea though, it's a good nit pick